composer.json

```diff
@@ -62,6 +62,7 @@
         "branch-alias": {
             "dev-master": "3.0.x-dev"
         },
+        "config-plugin-output-dir": "runtime/cache/config",
```

Comment:
How about runtime/build instead of runtime/cache? That is to give a hint that a build is required in order to get the data back if it's deleted.

Reply:
`runtime/build/config`?

Reply:
yes

Reply:
ok